spec/services/prison_visits/api_spec.rb

```diff
@@ -135,10 +135,6 @@
   describe 'request_visit', vcr: { cassette_name: 'request_visit' } do
     subject { super().request_visit(valid_booking_params) }
 
-    before do
```

Comment:
I think we do – you wrote a block to describe API localisation. However, they don't test the returned prison names to check that they're welsh, which the test probably should do. I can make the tests clearer later if you like?